modules/gdscript/gdscript.cpp

```diff
@@ -1994,8 +1994,9 @@ void GDScriptLanguage::get_reserved_words(List<String> *p_words) const {
 		"preload",
 		"signal",
 		"super",
-		"trait",
-		"yield",
+		"trait", // Not implemented yet, but reserved for future use. Highlight as a keyword to make errors easier to understand.
```

Comment:
I dont even get why this is here. no point

Reply:
I think vnen approved the idea of using traits in the past, see: #18698:

> * Re-use of functionality is also expected. So it needs constructs outside of inheritance to share code (such as Traits).